Source/player.cpp

```diff
@@ -1053,7 +1053,7 @@ void InitPlayer(int pnum, bool FirstTime)
 				player.position.tile = { ViewX, ViewY };
 			}
 		} else {
-			for (i = 0; i < 8 && !PosOkPlayer(pnum, plrxoff2[i] + player.position.tile.x, plryoff2[i] + player.position.tile.y); i++)
+			for (i = 0; i < 8 && !PosOkPlayer(pnum, Point { plrxoff2[i], plryoff2[i] } + player.position.tile); i++)

```

Comment:
Could the plr(x/y)off2 array also be converted to `Point` (in this or future pr)?

Reply:
I'm converting a lot of stuff. Eventually everything that _can_ use a `Point`, will. I'm just not doing everything on the same change to avoid huge PRs that stay open for great lengths of time and cause conflicts etc.

Doing incrementally avoids that.

Reply:
That's good and I like that approach. 👍 
I'm aware of this and I saw also other places that aren't converted (for example `ActivateSpawn`) and didn't comment.
I comment on this cause,
- both users of the array are modified by this pr (changed to `Point`).
- this kind of arrays can be forgotten.
- this is only a question/reminder. 😉

Reply:
Yeah I'm aware that this leaves some places "dangling" for a while but it's better than tackling everything in one go IMHO.